client/app/visualizations/edit-visualization-dialog.html

```diff
@@ -33,10 +33,18 @@ <h4 class="modal-title">Visualization Editor</h4>
       <div class="col-md-7" style="border: 1px solid #eee">
         <visualization-renderer visualization="$ctrl.visualization" query-result="$ctrl.queryResult"></visualization-renderer>
       </div>
+      <div class="col-md-7">  
+        <div class="bg-warning" ng-if="$ctrl.visualization.options.series.stacking == null && $ctrl.has3plusColumnsFunction()" style="padding:2%;" ng-bind-html="$ctrl.warning_three_column_stacking"> 
```

Comment:
Arik prefers creating a CSS file for each component and then requiring it since webpack will do the rest. Not a personal fan of importing CSS this way but we should stick to it for consistency. See https://github.com/getredash/redash/pull/1569/files#diff-855c4fc3efcfee2ac69a13e2f2810ecbR3 for reference.